jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -187,6 +187,7 @@ void ecma_value_assign_number (ecma_value_t *value_p, ecma_number_t ecma_number)
 void ecma_free_value (ecma_value_t value);
 void ecma_fast_free_value (ecma_value_t value);
 void ecma_free_value_if_not_object (ecma_value_t value);
+lit_magic_string_id_t ecma_get_typeof_lit_id (ecma_value_t value);
```

Comment:
One more minor thing: since this is not typeof, lets rename this function to `ecma_get_value_type_lit_id`.

Reply:
This function implements the typeof function as this was extracted from the op_typeof call. I'm using this in both places.